tests/subdir.am

```diff
@@ -3,6 +3,7 @@
 #
 
 PYTHON ?= python
+export ASAN_OPTIONS=detect_leaks=0
```

Comment:
Our `make` setup is non-recursive, this affects every command executed from the Makefile since it's included into the main one...